CONTRIBUTING.md

```diff
@@ -1,19 +1,24 @@
 Welcome!
+
 Every person creating InnerSource-related content or tools within their company should contribute to this working group.
+
 This working group helps all to get feedback on this work and to share it so that others can leverage it as well.
+
 Check out our [README.md] for more details.
 
-## 3 Ways to Contribute
+## Ways to Contribute
+
+```mermaid
+graph LR
+    WG[ISPO Working Group] --> KB[Kanban Board]
+    WG --> GD[Reviews]
```

Comment:
I removed the link to GitHub discussion from the Our Artifacts section because it's redundant.

We've successfully had discussions within the Kanban cards themselves. I think we should disable GitHub Discussions entirely to prevent confusion about where to discuss topics.

Reply:
I'm not an admin.

https://github.com/InnerSourceCommons/ispo-working-group

I created an issue from one discussion, but it didn't carry over all the content only the first post.

https://github.com/InnerSourceCommons/ispo-working-group/issues/68

I also couldn't find a way to export an issue out of the box.

Once I have admin, I'll find a way to export the discussions and then disable the discussions feature.